client/src/js/nav/components/__tests__/Bar.test.js

```diff
@@ -0,0 +1,47 @@
+import { Bar, mapStateToProps, mapDispatchToProps } from "../Bar";
+
+describe("<Bar />", () => {
+    const props = {
+        onGet: jest.fn(),
+        id: "foo",
+        administrator: true,
+        logout: jest.fn(),
+        pending: false
+    };
+    it("should render", () => {
+        const wrapper = shallow(<Bar {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("mapStateToProps", () => {
+    const state = {
+        account: { foo: "bar" },
+        app: {
+            pending: false
+        }
+    };
+    it("should return props", () => {
+        const result = mapStateToProps(state);
+        expect(result).toEqual({
+            foo: "bar",
+            pending: false
+        });
+    });
+});
+
+describe("mapDispatchToProps", () => {
+    let dispatch = jest.fn();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['dispatch' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4925540)